### PR TITLE
[#163385826] Add TouchID | FaceID icon to onboarding screen

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to add an icon to the fingerprint screen during onboarding process.

screen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#163385826](https://www.pivotaltracker.com/story/show/163385826): Come CIT, durante la fase di onboarding, se il dispositivo supporta il biometrico, voglio vedere uno screen che mi informa di questa possibilità

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: img/icons/faceid-onboarding-icon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="103" height="80" viewBox="0 0 103 80">
```
Comment (fpersico) [inline img/icons/faceid-onboarding-icon.svg]: You are including but not using the svg.

File: img/icons/fingerprint-onboarding-icon.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="66" height="80" viewBox="0 0 66 80">
```
Comment (fpersico) [inline img/icons/fingerprint-onboarding-icon.svg]: Same as above.

File: ts/screens/onboarding/FingerprintScreen.tsx [hunk 3]
```diff
@@ -62,14 +59,32 @@ export class FingerprintScreen extends React.PureComponent<Props, State> {
...
+   */
+  private renderIcon(biometryType: BiometrySimpleType) {
+    switch (biometryType) {
+      case "FACE_ID":
+        return require("../../../img/icons/faceid-onboarding-icon.png");
+      case "FINGERPRINT":
+      case "TOUCH_ID":
+      case "NOT_ENROLLED":
+      case "UNAVAILABLE":
+        return require("../../../img/icons/faceid-onboarding-icon.png");
```
[the remark this answers is not in the document]
Comment (lmorelli986) [reply ts/screens/onboarding/FingerprintScreen.tsx]: Sorry, an odd copy out.

File: locales/en/tos_fingerprint_enrolled.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+You can use {{biometryType}} in place of your PIN for accessing the app and for authorizing operations and payments in a faster and more sure way.
+
+Biometric information is saved on this device memory and will not communicated to IO's systems nor saved elsewehere at all.
+
+You can disable biometric authorization method whenever you want from this app security preference pane.
```
Comment (Undermaken) [inline locales/en/tos_fingerprint_enrolled.md]:
```suggestion
You can disable biometric authorization method whenever you want from this app security preference panel.
```

File: locales/it/tos_fingerprint_enrolled.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Potrai usare {{biometryType}} al posto del PIN per accedere all'app e autorizzare operazioni e pagamenti in modo più rapido e sicuro.
+
+Le informazioni biometriche vengono salvate nella memoria del tuo telefono, non verranno in nessun modo comunicate ai sistemi di IO o salvate da altre parti.
+
+Potrai disattivare il metodo di autenticazione biometrico in qualunuque momento dalle preferenze dell'applicatzione
```
Comment (Undermaken) [inline locales/it/tos_fingerprint_enrolled.md]:
```suggestion
Potrai disattivare il metodo di autenticazione biometrico in qualunque momento dalle preferenze dell'applicazione
```

Comment (lmorelli986) [reply locales/it/tos_fingerprint_enrolled.md]: thx. fixed.

File: locales/en/tos_fingerprint_enrolled.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+You can use {{biometryType}} in place of your PIN for accessing the app and for authorizing operations and payments in a faster and more sure way.
```
[the remark this answers is not in the document]
Comment (lmorelli986) [reply locales/en/tos_fingerprint_enrolled.md]: changed

File: locales/en/tos_fingerprint_enrolled.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+Biometric information is saved on this device memory and will not communicated to IO's systems nor saved elsewehere at all.
```
[the remark this answers is not in the document]
Comment (lmorelli986) [reply locales/en/tos_fingerprint_enrolled.md]: changed

File: locales/en/tos_fingerprint_enrolled.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lmorelli986) [reply locales/en/tos_fingerprint_enrolled.md]: changed

File: locales/en/tos_fingerprint_not_enrolled.md [hunk 8]
```diff
@@ -0,0 +1 @@
+No enabled biometric authorization method has been found. If you want you can formerly enable it on this device from settings section, and then enable it from this app security preference pane.
```
[the remark this answers is not in the document]
Comment (lmorelli986) [reply locales/en/tos_fingerprint_not_enrolled.md]: changed

**Pull request merged**